### ISSUE TITLE
How do I disable form support in order to show unverified signatures?

### ISSUE THREAD

**Issue by tlarrain** [stephanrauh/ngx-extended-pdf-viewer#577]
I'm trying to show unverified signatures on my pdfs by using `[showUnverifiedSignatures]="true"` but it's still not showing anything. I saw in the documentation that I "have to disable form support", but I didn't find anything about that in the documentation. Any help is welcomed!

**Comment by stephanrauh** [comment]
Sounds like I ought to improve the documentation. Which documentation are you referring to, precisely?

Disabling form support is simply a matter of setting a default option before loading the PDF file, like so:

```typescript
@Component({ ... })
export class SignatureComponent {
  constructor() {
    pdfDefaultOptions.renderInteractiveForms = false; // <<<< disables form support
  }
}
```

There's a live demo at https://pdfviewer.net/signatures.

**Comment by stephanrauh** [comment]
I'm closing the ticket because I believe my answer helps you, but I'd appreciate your feedback. Did you manage to solve the issue?

Best regards,
Stephan

**Comment by tlarrain** [comment]
Thanks for your answer. I was looking at the live sample and I also inspected the code, but I can't see where to bind those `pdfDefaultOptions`. Is it a property of the component? How do I associate it with the component? I tried to use `[renderInteractiveForms]="false"` but it didn't work. I also tried to use a `@ViewChild`, but the property doesn't seem to exist. I would appreciate a little more detail on where to set that property.
